app/assets/css/sass/rtmedia.scss

```diff
@@ -39,3 +39,6 @@
 
 // *** RTL *** //
 @import "rtl";
+
+// *** BP Media Message *** //
+@import "media-message";
```

Comment:
i don't see any **scss** file for this `@import "media-message";`
if not used please remove.